### PR TITLE
feat(ConferenceInfo): 🚀 improve the ConferenceInfo component to display different amount of avatars when the window resizing.  #RCINT-8622

### PR DESCRIPTION
**Pull request by RukaiYu**
#RCINT-8622

### CODE REVIEW

File: packages/ringcentral-widgets/components/ActiveCallPanel/ConferenceInfo.js [hunk 1]
```diff
@@ -1,14 +1,110 @@
...
+const AVATAR_MERGIN_LEFT = parseInt(styles.avatarMerginLeftSize, 0); // -20
+const PEDDING_WIDTH = parseInt(styles.avatarPaddingSize, 0); // 15
+
+const minWidthCalculator = count =>
+  (WIDTH_PER_AVATAR * count) +
+  AVATAR_MERGIN_LEFT * (count - 1) + (PEDDING_WIDTH * 2) + 1 + 2;
+
+// when the container width reachs below item of width, display the avatar amount of count.
+const KINDS_OF_WIDTH_THAT_NEED_ADAPATER = [
+  { avartarCount: 0, width: minWidthCalculator(2), },
```
Comment (huanhulan) [inline packages/ringcentral-widgets/components/ActiveCallPanel/ConferenceInfo.js]: we show at least 1 avatar

File: packages/ringcentral-widgets/components/ActiveCallPanel/ConferenceInfo.js [hunk 2]
```diff
@@ -1,14 +1,110 @@
...
+  onWindowResize = debounce(() => {
+    this.updateAvatarAmounts(this.props);
+  }, 100);
+
+  updateAvatarAmounts(props) {
+    if (!this._mounted) {
+      return;
+    }
+
+    const avatarCount = this._computeAvatarCountByWindowWidth(props);
```
Comment (huanhulan) [inline packages/ringcentral-widgets/components/ActiveCallPanel/ConferenceInfo.js]: if `avatarCount ` equals to `this.state. avatarCount `, we don't need the further calculations

File: packages/ringcentral-widgets/components/ActiveCallPanel/ConferenceInfo.js [hunk 3]
```diff
@@ -25,53 +121,74 @@ export class ConferenceInfo extends Component {
...
-    const displayedProfiles =
-      profiles.length >= MAXIMUM_AVATARS
-        ? profiles.slice(0, MAXIMUM_AVATARS)
-        : profiles;
+    const { avatarCount } = this.state;
 
-    const remains = profiles.length > MAXIMUM_AVATARS
-      ? profiles.length - MAXIMUM_AVATARS
-      : 0;
+    const isAvatarCountComputingReady = !!avatarCount || avatarCount === 0;
```
Comment (huanhulan) [inline packages/ringcentral-widgets/components/ActiveCallPanel/ConferenceInfo.js]: remove this one

File: packages/ringcentral-widgets/components/ActiveCallPanel/styles.scss [hunk 4]
```diff
@@ -3,14 +3,19 @@
 @import '../../lib/commonStyles/fonts.scss';
 @import '../../lib/commonStyles/layout';
 $userInfoHeight: 134px;
+$conferenceAvatarSizeInt: 51;
 $conferenceAvatarSize: 51px;
+$avaterMerginLeftSizeInt: -20;
+$avaterMerginLeftSize: -20px;
```
Comment (huanhulan) [inline packages/ringcentral-widgets/components/ActiveCallPanel/styles.scss]: $avaterMerginLeftSize: $avaterMerginLeftSizeInt + unquote("px");

File: packages/ringcentral-widgets/components/ActiveCallPanel/ConferenceInfo.js (hunk 1 above)
Comment (RukaiYu) [reply packages/ringcentral-widgets/components/ActiveCallPanel/ConferenceInfo.js]: Confirmed with Albert, we could show it like that when the window is at <100 widths.

**Pull request merged**